operations/sales/partner-programs.md

```diff
@@ -1,5 +1,7 @@
 # Partner Programs
 
+**Are you an existing customer interested in becoming a partner?** Reach out to your Customer Success Manager to explore partnership opportunities.
```

Comment:
How should they conctact the CSM?

Reply:
Good question, I'm not sure about the customer journey here, I'll find out and report back by next Monday.